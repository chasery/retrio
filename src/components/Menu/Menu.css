.Menu {
  position: absolute;
  top: 3rem;
  right: 0;
  z-index: 10;
  visibility: hidden;
  min-width: 12rem;
  padding: 0.25rem 0;
  box-shadow: 0 0.25rem 0.25rem #0000004c;
  background-color: #fff;
}
.Menu.visible {
  visibility: visible;
}

.Menu > li {
  display: flex;
  font-size: 1rem;
  font-weight: normal;
  line-height: auto;
}
.Menu > li + li {
  border-top: 0.0625rem solid #ebebeb;
}

.Menu > li > a {
  flex-grow: 1;
  padding: 1rem;
}
