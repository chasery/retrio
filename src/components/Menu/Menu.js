import React from 'react';
import './Menu.css';

function Menu(props) {
  const { visible, children } = props;

  return <ul className={`Menu ${visible ? 'visible' : ''}`}>{children}</ul>;
}

export default Menu;
