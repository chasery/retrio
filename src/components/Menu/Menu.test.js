import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Menu from './Menu';

describe('Menu component', () => {
  it('renders without error', () => {
    const wrapper = shallow(<Menu />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
