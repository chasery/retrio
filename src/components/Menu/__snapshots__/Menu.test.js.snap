// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu component renders without error 1`] = `
<ul
  className="Menu "
/>
`;
